### ISSUE TITLE
neuralynxio.py memmap possibly float instead of int

### ISSUE THREAD

**Issue by sin-mike** [NeuralEnsemble/python-neo#393]
in code of memmapping the division could give floats instead of ints even if the result is even
```
data = np.memmap(self.sessiondir + sep + filename,
                                   dtype='<u2',
                                   shape=((filesize - 16384) / 2 / 56, 56),
                                   mode='r', offset=16384)
```

a simple workaround is to force it to int conversion (at least works in my case)

```
data = np.memmap(self.sessiondir + sep + filename,
                                  dtype='<u2',
                                  shape=(int((filesize - 16384) / 2 / 56), 56),
                                   mode='r', offset=16384)

```

**Comment by samuelgarcia** [comment]
Thank.

I guess this is already fixed in RP #385.

**Comment by sin-mike** [comment]
@samuelgarcia 
Indeed! Thank you, then I must change to the development version rather than PyPi
